src/main.cpp

```diff
@@ -731,7 +731,7 @@ int main ( int argc, char** argv )
 \******************************************************************************/
 QString UsageArguments ( char** argv )
 {
-    return "Usage: " + QString ( argv[0] ) +
+    return "\nUsage: " + QString ( argv[0] ) +
```

Comment:
I don't like the adding of leading `\n` in various places.  Why here, at all, too?

Reply:
Had the same question, but @henkdegroot explains it here:

>  Also add linebreak to the start of the text so in windows cmd the text start at a new line.

Reply:
Qt should fix their Windows support so we're still attached to the cmd session...  (Or we display the usage in a window for MSWindows - that's a common approach, too.)

Reply:
> Or we display the usage in a window for MSWindows - that's a common approach, too.

Indeed a common approach, but won't resolve it when you use a Windows Server without Destop :)
In #1660 it is suggested to use QCommandLineParser...however don't think that will change the behaviour in this case as it states:
_On Windows, QCommandLineParser uses message boxes to display usage information and errors if no console window can be obtained._